### PR TITLE
added num 3 to variables list

### PR DESCRIPTION
**Pull request by ideyology**
just adding another variable thanks

### CODE REVIEW

**Comment by ideyology** [comment]
parang no eh

**Review by arcortez — CHANGES_REQUESTED** [review]
di mo naman yan kailangan tanggalin mo na

File: CORTEZ_MinorA/Form1.vb [hunk 1]
```diff
@@ -7,6 +7,7 @@
     Dim LastResult As Double
     Dim IsNewOperation As Boolean
     Dim LastNum2Value As Double
+    Dim LastNum3Value As Double
```
Comment (arcortez) [inline CORTEZ_MinorA/Form1.vb:10]: issue here. fix pls.

Comment (ideyology) [reply CORTEZ_MinorA/Form1.vb:10]: thi is fine

**Review by arcortez — APPROVED** [review]

**Pull request closed without merging**